### PR TITLE
Subset of metadata fields

### PR DESCRIPTION
**Pull request by rsevilla87**
## Type of change

- [ ] Refactor
- [x] New feature
- [ ] Bug fix
- [ ] Optimization
- [ ] Documentation Update

## Description

Follow-up of https://github.com/kube-burner/kube-burner/pull/702
As agreed with the team (cc @mukrishn), let's add a small subset of metadata fields to all metrics (ocpVersion and ocpMajorVersion) generated by kube-burner.

## Related Tickets & Documents

- Related Issue #
- Closes #

## Checklist before requesting a review

- [ ] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -85,3 +85,5 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.4.1 // indirect
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
+
+replace github.com/kube-burner/kube-burner => github.com/rsevilla87/kube-burner v1.7.6-0.20240926073724-2b0caacee619
```
Comment (rsevilla87) [inline go.mod:89]: This reference must be changed before merging

File: go.mod [hunk 2]
```diff
@@ -85,3 +85,5 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.4.1 // indirect
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
+
+replace github.com/kube-burner/kube-burner => github.com/rsevilla87/kube-burner v1.7.6-0.20240926090502-2e59036445b9
```
Comment (vishnuchalla) [inline go.mod:89]: upstream is merged, will wait for this to be updated as well.

**Review by venkataanil — APPROVED** [review]
LGTM

**Review by venkataanil — APPROVED** [review]
LGTM

**Pull request merged**